src/execution/sql/ddl_executors.cpp

```diff
@@ -152,8 +153,21 @@ bool DDLExecutors::CreateIndex(const common::ManagedPointer<catalog::CatalogAcce
   storage::index::IndexBuilder index_builder;
   index_builder.SetKeySchema(schema);
   auto *const index = index_builder.Build();
-  bool result UNUSED_ATTRIBUTE = accessor->SetIndexPointer(index_oid, index);
-  TERRIER_ASSERT(result, "CreateIndex succeeded, SetIndexPointer must also succeed.");
+
+  if (concurrent) {
+
```

Comment:
hmm is the concurrent case not implemented as of now?

Reply:
We'd like to keep the concurrent variables since we're using them to tell the user CREATE INDEX CONCURRENTLY is not implemented. Plus, putting them in is an annoying process and they dont do any harm sitting there waiting for someone to implement CONCURRENTLY